resources/deployment.yaml

```diff
@@ -34,11 +34,11 @@ spec:
     spec:
       serviceAccountName: objectstorage-provisioner-sa
       containers:
-      - name: objectstorage-sample-driver
-        image: quay.io/containerobjectstorage/objectstorage-sample-driver:latest
+      - name: minio-cosi-driver
+        image: quay.io/containerobjectstorage/minio-cosi-driver:latest
         args:
-          - "--listen-address=$(LISTEN_ADDRESS)"
-          - "--s3-endpoint=$(S3_ENDPOINT)"
+          - "--driver-addr=$(DRIVER_ADDRESS)"
+          - "--endpoint=$(ENDPOINT)"
```

Comment:
The variables endpoint, access-key, secret-key are not used in the driver and the pod fails to start since it doesn't expect that flag

Reply:
yeah, you should not deploy wiht old kustomize template

Reply:
yeah driver update is still WIP